client/keys/add.go

```diff
@@ -200,6 +195,11 @@ func runAddCmd(_ *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
+
+		if !bip39.IsMnemonicValid(mnemonic) {
+			_, _ = fmt.Fprintf(os.Stderr, "Error: Mnemonic is not valid.\n")
```

Comment:
If you really want to return an error (as in: exiting the program with return code != 0) then this should be:

```golang
return errors.New("invalid mnemonic")
```

Reply:
Actually, yeah this is preferable @yangyanqing.

Reply:
Thank you for correction ! @alessio @alexanderbez